backend/Task.py

```diff
@@ -0,0 +1,34 @@
+from flask import jsonify
+class Task():
+
```

Comment:
I like the idea of creating a class for the Tasks. It's a good way to store the data of task and I think I'll do this in the front end as well.

Reply:
Facts